spec/features/work_packages/work_package_index_spec.rb

```diff
@@ -62,7 +62,7 @@
       visit project_path(project)
 
       within("#content") do
-        expect(page).to have_content("Overview")
+        expect(page).to have_heading "Overview"
```

Comment:
N.B. this will now fail when the feature flag is eventually removed - `have_content` was previously matching the tab content.

Reply:
Maybe add that as a comment as well?